pkg/i18n/traditional_chinese.go

```diff
@@ -3,54 +3,53 @@
 The selection of Traditional Chinese translation vocabulary is mainly based on
 the following sources:
 
- 1. GitLab: 其介面有相當完整的繁體中文翻譯，但缺少一些本地端功能的對照，例如 stash。
+1. GitLab: 其介面有相當完整的繁體中文翻譯，但缺少一些本地端功能的對照，例如 stash。
 
- 2. Pro Git: Git 的權威參考用書，可惜繁中部分翻譯僅約一半。
-    https://git-scm.com/book/zh-tw/v2
+2. Pro Git: Git 的權威參考用書，可惜繁中部分翻譯僅約一半。
+https://git-scm.com/book/zh-tw/v2
 
- 3. Microsoft 語言入口網站 (Visual Studio)
-    https://www.microsoft.com/zh-tw/language/
+3. Microsoft 語言入口網站 (Visual Studio)
+https://www.microsoft.com/zh-tw/language/
 
 ### Glossary ###
 
-	譯文中括號內文字會依語境添加或省略。
+譯文中括號內文字會依語境添加或省略。
 
-	Repository		版本庫
-	Amend			修正
-	Checkout		檢出
-	Cherry-pick		揀選
-	Diff			差異
-	Discard			捨棄
-	Drop [stash]	捨棄
-	Fast-forward	快進式 (Fast-forward)
-	Fetch			擷取
-	Fixup			修復 (Fixup)
-	Patch			補丁
-	Pop [stash]		還原
-	Rebase			變基 (Rebase)
-	Reset			重設
-	Revert			還原
-	Reword			改寫
-	Squash			壓縮 (Squash)
-	Stage			預存 (Stage)
-	Stash			收藏 (Stash)
+Repository		版本庫
+Amend			修改
+Checkout		檢出
+Cherry-pick		揀選
+Diff			差異
+Discard			捨棄
+Drop [stash]		捨棄
+Fast-forward		快轉 (Fast-forward)
+Fetch			擷取
+Fixup			修復 (Fixup)
+Patch			補丁
+Pop [stash]		還原
+Rebase			變基 (Rebase)
+Reset			重設
+Revert			還原
+Reword			改寫
+Squash			壓縮 (Squash)
+Stage			預存 (Stage)
+Stash			收藏 (Stash)
 */
 package i18n
 
 const traditionalChineseIntroPopupMessage = `
-感謝你使用 lazygit，你真的很厲害。有三件事要與你分享：
+感謝使用 lazygit！這裡有一些資源可供參考：
 
- 1) 如果你想了解 lazygit 的功能，請看這個影片：
-      https://youtu.be/CPLdltN7wgE
+1) 📺lazygit 教學📺：
+https://youtu.be/CPLdltN7wgE
 
- 2) 請務必閱讀最新的發布說明：
-      https://github.com/jesseduffield/lazygit/releases
+2) 📣釋出說明📣：
+https://github.com/jesseduffield/lazygit/releases
 
- 3) 如果你正在使用 git，那麼就是一個程式設計師！在你的幫助下，
-	我們可以使 lazygit 變得更好，請考慮成為貢獻者並一起同樂：
-      https://github.com/jesseduffield/lazygit
-	你也可以贊助我，並告訴我有什麼要改進的，點擊右下方的捐贈按鈕即可。
-	就算是只在版本庫中點個星星分享愛也很棒！
+3) 💖如果你想要貢獻一份心力你可以💖：
+改進 lazygit 原始碼：https://github.com/jesseduffield/lazygit
+按右下角的捐款斗內我們
+或單存添加 lazygit 到你的 star 清單內以增加曝光度都能大力的幫助我們！
```

Comment:
The indentation of this message has changed, is there a reason for it? I suppose you have some auto-formatter that did this automatically. I see that the indentation of the glossary above has also changed.

Reply:
No I don't. I thought that the indentation isn't necessary. Should I change it back

Reply:
Yes, I think it would be good to format it in the same way as the english one, unless there are some language rules in Chinese that speak against it.

Reply:
is it good now

Reply:
Well, not really, the commit history is still a mess containing merge commits that are impossible to understand, and the file was not auto-formatted correctly. I fixed all this in #3394 as a draft PR, and added a commit to change the indentation back to how it was before; you could now either `git reset --hard` onto that one and force-push, or we just merge mine. I don't care very much either way.

I do have to say that I found this experience a bit painful, and I don't really like spending time on things like this. On the other hand, I value your contribution and I didn't want your work to be lost, so thanks for your work.